CHANGES.txt

```diff
@@ -1,4 +1,6 @@
+<<<<<<< HEAD
```

Comment:
Nit: this seems a leftover of solving merge conflicts

Reply:
Done and committed